modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,50 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
```

Comment:
```suggestion
{product-title} routes support the use of custom labels and annotations, which you can configure by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
```

An attempt at making this more active.

Reply:
Also, should "Knative ingress" be something like "Knative ingress gateway?"

Reply:
Waiting for a response from eng about the ingress thing

Reply:
I think "Knative ingress" is better in this case.

In the sentence, the "Knative ingress" means the name of custom resource (`ingresses.networking.internal.knative.dev`) and we usually call it `knative ingress` or `kingres`, `king`.